crates/factor-variables/build.rs

```diff
@@ -1,5 +1,5 @@
 fn main() {
-    println!("cargo::rerun-if-changed=build.rs");
+    println!("cargo:rerun-if-changed=build.rs");
```

Comment:
If this line necessary? I believe I commented before that build.rs files automatically build if they're modified, but I'm not sure what the response to that was.

Reply:
Ah - I had somehow completely forgotten that that's the case...